File: no-auth/src/main/java/com/kjj/noauth/dto/keycloak/KeycloakUserInfoDto.java
package com.kjj.noauth.dto.keycloak;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@AllArgsConstructor
@NoArgsConstructor
public class KeycloakUserInfoDto {
    private String sub;
    private boolean emailVerified;
    private String[] roles;
    private String name;
    private String preferredUsername;
    private String givenName;
    private String familyName;
}

File: no-auth/src/main/kotlin/com/kjj/noauth/dto/keycloak/KeycloakUserInfoDto.kt
package com.kjj.noauth.dto.keycloak

data class KeycloakUserInfoDto(
    val sub: String,
    val emailVerified: Boolean,
    val roles: Array<String>,
    val name: String,
    val preferredUsername: String,
    val givenName: String,
    val familyName: String
) {
}